nixui/graphics/color_indicator.py

```diff
@@ -10,11 +10,12 @@ def get_edit_state_color_indicator(tree, option_path):
     configured_definition exists: yellow
     system_default_definition: white
     """
+    # TODO use icons to diff memory/configuration definition change
     # in memory definition change
     if option_path in tree.get_change_set_with_ancestors():
-        return QtGui.QColor(214, 253, 221)  # light green
+        return QtGui.QPalette().color(QtGui.QPalette().Highlight)
```

Comment:
Is there a `QPalette` color we can use that is distinct from color for the currently selected item?

Reply:
there is something like "alternating row color" for qt list widgets
usually alternating between white and lightgrey

Reply:
I've tried this and it doesn't seem to work. I'll work off of your changes and remove all coloring, and instead use icons to represent edit state as you suggested.

Reply:
why blue and green?
what exactly is "in memory definition change" vs "in configuration definition change"?

pencil? could be more abstract  ... like codium just shows a bullet (full circle) for changed files

Reply:
Yes, circle works well I think.

Blue and green because I'm not sure of better colors that would work well with light mode and dark mode.

In memory = changed in Nix-Gui, but not committed to `configuration.nix` (nor to a module in `imports`) via the Save button. 

In configuration = part of `configuration.nix`.

Reply:
mkay. how about

\+ = add config

&ndash; = remove config

● = active config, existing config, old config, currently active system config

○ = disabled config, potential config, as allowed by nixos

add/remove config = "in memory", not-yet committed, "staged" for commit &rarr; manage config with git